src/REST/ClientRaw.php

```diff
@@ -40,17 +40,18 @@ class ClientRaw
 
     public static function instance() : ClientRaw
     {
-        if (empty(self::$instance)) {
-            self::$instance = new self();
+        if (empty(static::$instance)) {
+            static::$instance = new self();
```

Comment:
What do you think about using `new static()` instead of `new self()` here as well?

I'm not a big fan of `new static` by the way. Strictly speaking, we should to make the class or its constructor `final`, if we are using `new static`. Otherwise, derived classes will have no option to add arguments to their own constructors, for instance.